AppInspector/Commands/AnalyzeCommand.cs

```diff
@@ -274,6 +274,7 @@ private void ConfigRules()
 
             //instantiate a RuleProcessor with the added rules and exception for dependency
             _rulesProcessor = new RuleProcessor(rulesSet, _confidence, !_options.AllowDupTags, _options.MatchDepth == "first", _options.Log);
+            _rulesProcessor.UniqueTagExceptions = "Metric.".Split(",");//fix to enable non-unique tags if metric counter related
```

Comment:
This line just sets unique tag exceptions to an array of strings which only contains "Metric." as its first entry.